---
include:
  xacro: digital_twin_params_mk1.yaml
  scaler_consts: scaler_consts_mk1.yaml
  agevar_consts: agevar_consts_mk1.yaml
version: "mk1"
canbus:
  channel: "can0"
num_modules: 2
modules:
  - address: 0x11
    hasJoint: false
    hasEndEffector: true
  - address: 0x12
    hasJoint: true
    hasEndEffector: false
realsense_config: "realsense_rgb_motion.yaml"
sensors:
  - realsense: true